tests/utils/test_files.py

```diff
@@ -9,3 +9,46 @@ def test_get_file_in_directory(nested_directory_with_file: Path, some_file: str)
     file_path: Path = get_file_in_directory(nested_directory_with_file, some_file)
     # THEN assert that the file is returned
     assert file_path.exists()
+
+
+def test_rename_file(tmp_path: Path):
+    # GIVEN a file path and a renamed file path
+    file_path: Path = Path(tmp_path, "dummy_path")
+    renamed_file_path: Path = Path(tmp_path, "dummy_renamed_path")
+
+    # GIVEN that the file path exist
+    file_path.touch()
+    assert file_path.exists()
+
+    # GIVEN that the renamed file path does not exist
+    assert not renamed_file_path.exists()
+
+    # WHEN renaming the file
+    rename_file(file_path=file_path, renamed_file_path=renamed_file_path)
+
+    # THEN the renamed file path should exist
+    assert renamed_file_path.exists()
+
+    # THEN the file path should not exist
+    assert not file_path.exists()
+
+
+def test_rename_file_exists(tmp_path: Path, caplog):
+    # GIVEN a file path and a renamed file path
+    caplog.set_level("DEBUG")
+    file_path: Path = Path(tmp_path, "dummy_path")
+    renamed_file_path: Path = Path(tmp_path, "dummy_renamed_path")
+
+    # GIVEN that the file path exist
+    renamed_file_path.touch()
+    assert renamed_file_path.exists()
+
+    # WHEN renaming the file
+    rename_file(file_path=file_path, renamed_file_path=renamed_file_path)
+
+    # THEN the renamed file path should exist and a debug message should be logged
+    assert renamed_file_path.exists()
+    assert (
+        f"File {renamed_file_path} already exists. Skipping renaming of {renamed_file_path}."
+        in caplog.text
+    )
```

Comment:
Consider adding a test case for what happens if a fastq file already has been renamed